fabric-data-fixer-api-v1/src/main/resources/fabric.mod.json

```
@@ -0,0 +1,41 @@
+{
+  "schemaVersion": 1,
+  "id": "fabric-data-fixer-api-v1",
+  "name": "Fabric Data Fixer API (v1)",
+  "version": "${version}",
+  "environment": "*",
+  "license": "Apache-2.0",
+  "icon": "assets/fabric-data-fixer-api-v1/icon.png",
+  "contact": {
+    "homepage": "https://fabricmc.net",
+    "irc": "irc://irc.esper.net:6667/fabric",
+    "issues": "https://github.com/FabricMC/fabric/issues",
+    "sources": "https://github.com/FabricMC/fabric"
+  },
+  "authors": [
+    "FabricMC",
+    "QuiltMC"
```

```suggestion
    "FabricMC"
```
We haven't marked individual contributors as authors either, so I think just FabricMC is fine